Error when building workflow with openapi spec with multipart content

### Describe the bug

With 1.34.0 when building I have the following error:

> [ERROR] /home/kogito/serverless-workflow-project/target/generated-sources/open-api-stream/org/kie/kogito/openapi/movekube/api/WorkspaceInputsApi.java:[61,60] package org.jboss.resteasy.annotations.providers.multipart does not exist


See enclosed the openapi spec file
[move2kube.zip](https://github.com/user-attachments/files/17364723/move2kube.zip)


### Expected behavior

No issue when building the workflow

### Actual behavior

Error

### How to Reproduce?

1. Create a workflow
2. Use the enclosed spec or any spec with multipart and/or application/x-www-form-urlencoded content
3. Build the workflow using OSl 1.34 (registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel8@sha256:9a4093e195bec163c609381e3c0ceeec55f6e229c6a1def6f362517886faea71)
4. See the error

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Kogito version or git rev (or at least Quarkus version if you are using Kogito via Quarkus platform BOM)

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/quarkus/extensions/kogito-quarkus-serverless-workflow-extension/kogito-quarkus-serverless-workflow/pom.xml
+++ b/quarkus/extensions/kogito-quarkus-serverless-workflow-extension/kogito-quarkus-serverless-workflow/pom.xml
@@ -59,22 +59,22 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-rest-client</artifactId>
     </dependency>
-
     <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-rest-client-jackson</artifactId>
     </dependency>
-
     <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-smallrye-openapi</artifactId>
     </dependency>
-
     <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-grpc</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-multipart-provider</artifactId>
+    </dependency>
     <!-- Support CloudEvents OotB -->
     <dependency>
       <groupId>org.kie</groupId>

